kolibri/plugins/media_player/assets/src/views/MediaPlayerTranscript/TranscriptCue.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+
+  <a
+    role="button"
+    tabindex="0"
+    :class="[
+      'transcript-cue',
+      { active },
+      $computedClass(style)
+    ]"
+    :dir="dir"
+    :title="$tr('title', { startTime })"
+    :aria-current="active.toString()"
+    @click="triggerSeekEvent"
+    @keypress.enter="triggerSeekEvent"
+  >
+    <span
+      class="transcript-cue-time"
+      :aria-label="$tr('timeLabel')"
+      :style="timeStyle"
+    >{{ startTime }}</span>
+    <span
+      class="transcript-cue-text"
+      :aria-label="$tr('textLabel')"
+      :style="textStyle"
+    >
+      <strong v-if="speaker">{{ speaker }}</strong>
+      {{ text }}
+    </span>
+  </a>
+
+</template>
+
+
+<script>
+
+  import videojs from 'video.js';
+  import themeMixin from 'kolibri.coreVue.mixins.themeMixin';
+  import { getLangDir } from 'kolibri.utils.i18n';
+
+  const SPEAKER_REGEX = /<v ([^>]+)>/i;
+
+  export default {
+    name: 'TranscriptCue',
+
+    mixins: [themeMixin],
+
```

Comment:
Gotcha. I would like to say that, whitespace is not an enemy! I've found most of the Kolibri code super dense. I once used the same style that LE does for JS, but after trying Allman style, which produces more visual whitespace, found it so much more readable. Just my preference and thoughts!

Reply:
That's reasonable – I don't feel strongly on the matter myself. I would just recommend that we systematically change it everywhere rather than have multiple styles in the same codebase.